sonic_api.yaml

```diff
@@ -1400,3 +1518,9 @@ definitions:
       ip_addr:
         type: string
         description: IP address of directly attached device to the switch
+  VrfEntry:
+    type: object
+    properties:
+      rd:
+        type: string
+        description: route distinguisher for this VRF to be added to routes in this VRF
```

Comment:
what is this used for?

Reply:
Discussed internally and looks like we dont need it (atleast for now). I will remove the VrfEntry object altogether and bring it back in when we extend the vrf object with other necessary fields.